freesound/static/bw-frontend/src/pages/front.js

```diff
@@ -1,18 +1,43 @@
-import throttle from 'lodash.throttle';
-import navbar from '../components/navbar';
+import './page-polyfills'
+import throttle from 'lodash.throttle'
+import navbar from '../components/navbar'
+import { addTypeAheadFeatures } from '../components/typeahead'
+import wait from '../utils/wait'
 
-const heroSearch = document.getElementsByClassName('bw-front__hero-search')[0];
+const heroSearch = document.getElementsByClassName('bw-front__hero-search')[0]
 
-const SCROLL_CHECK_TIMER = 100; // min interval (in ms) between consecutive calls of scroll checking function
+const input = document.getElementById('search-sounds')
+
+const SCROLL_CHECK_TIMER = 100 // min interval (in ms) between consecutive calls of scroll checking function
 const checkShouldShowSearchInNavbar = throttle(() => {
-  const heroRect = heroSearch.getBoundingClientRect();
+  const heroRect = heroSearch.getBoundingClientRect()
   // not all browsers support clientRect.height
-  const heroSearchPosition = heroRect.height ? heroRect.y + heroRect.height : heroRect.y;
-  const shouldShowSearchBar = heroSearchPosition < 80;
-  const isShowingSearchBar = !navbar.classList.contains('bw-nav--expanded');
+  const heroSearchPosition = heroRect.height
+    ? heroRect.y + heroRect.height
+    : heroRect.y
+  const shouldShowSearchBar = heroSearchPosition < 80
+  const isShowingSearchBar = !navbar.classList.contains('bw-nav--expanded')
   if (shouldShowSearchBar !== isShowingSearchBar) {
-    navbar.classList.toggle('bw-nav--expanded');
+    navbar.classList.toggle('bw-nav--expanded')
   }
-}, SCROLL_CHECK_TIMER);
+}, SCROLL_CHECK_TIMER)
+
+window.addEventListener('scroll', checkShouldShowSearchInNavbar)
+
+const fetchSuggestions = async query => {
```

Comment:
This is the function that you'll need to change in order for it to fetch actual suggestions from your BE